app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/cloud_database.rb

```diff
@@ -1,6 +1,21 @@
 class ManageIQ::Providers::IbmCloud::VPC::CloudManager::CloudDatabase < ::CloudDatabase
+  supports :create
   supports :delete
 
+  def self.raw_create_cloud_database(ext_management_system, options)
+    ext_management_system.with_provider_connection do |connection|
+      resource_group = connection.resource.manager.collection(:list_resource_groups).find { |resource| resource[:name] == options[:resource_group_name] }
```

Comment:
Included an extra step to get the resource group from the name here and then later use it to get the ID  as opposed to just getting the user to enter the ID directly because the resource group ID isn't readily available on the IBM Cloud UI.